components/brave_shields/browser/ad_block_base_service.h

```diff
@@ -45,6 +46,9 @@ class AdBlockBaseService : public BaseBraveShieldsService {
   void EnableTag(const std::string& tag, bool enabled);
   bool TagExists(const std::string& tag);
 
+  adblock::HostnameResources hostnameCosmeticResources(const std::string& hostname);
```

Comment:
I'm a bit confused by this, it looks like the implementation of this function just calls directly into the return type which seems to come from wrapper.hpp. But from what I can tell it should return an `std::string` and not `HostnameResources` here https://github.com/brave/adblock-rust-ffi/pull/7/files#diff-afc8c31c40724bcf2f02d090d196dbedR60

I don't even see `HostnameResources` defined anywhere. I must be missing something, or else there's a missing commit or something.  Please explain ,thanks!

Reply:
Sorry, that didn't get updated since changing the FFI interface. You are correct in that it should be a `std::string`.